.pre-commit-config.yaml

```diff
@@ -0,0 +1,18 @@
+exclude: ^(src/blib2to3/|profiling/|unit_tests/data/)
+
+repos:
+  - repo: https://github.com/pycqa/pylint
+    rev: pylint-2.6.0
+    hooks:
+      - id: pylint
```

Comment:
This needs to be the same pylint we run in our GitHub actions since pylint returns non-0 codes on anything.

It also doesn't seem to run in the virtualenv, which means it will just never work correctly for me